Modules/arraymodule.c

```diff
@@ -2989,6 +2989,26 @@ array_modexec(PyObject *m)
         Py_DECREF((PyObject *)&Arraytype);
         return -1;
     }
+
+    PyObject *abc_mod = PyImport_ImportModule("collections.abc");
+    if (!abc_mod) {
+        Py_DECREF((PyObject *)&Arraytype);
+        return -1;
+    }
+    PyObject *mutablesequence = PyObject_GetAttrString(abc_mod, "MutableSequence");
+    Py_DECREF(abc_mod);
+    if (!mutablesequence) {
+        Py_DECREF((PyObject *)&Arraytype);
+        return -1;
+    }
+    PyObject *res = PyObject_CallMethod(mutablesequence, "register", "O", (PyObject *)&Arraytype);
```

Comment:
```suggestion
    _Py_IDENTIFIER(register);
    PyObject *res = _PyObject_CallMethodIdOneArg(mutablesequence, &PyId_register, (PyObject *)&Arraytype);
```

Reply:
Thanks, @brandtbucher. The reason I originally did not add `_Py_IDENTIFIER` is so we don't add more to the global/static state so it does not make future sub interpreters isolation harder, especially given that this code will run only at module import time.